mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/print/config.yaml

```diff
@@ -37,7 +37,7 @@ scales:
   - 559082264
 
 # see https://github.com/georchestra/georchestra/issues/362
-brokenUrlPlaceholder: http://sdi.georchestra.org/public/transparent.gif
+brokenUrlPlaceholder: https://sdi.georchestra.org/public/transparent.gif

```

Comment:
Something funny about this: In my docker composition, since I use the latest available official image from jetty (which derives from official openjdk one), the Java version seems recent enough :) (But I have some problems with the JVM used on my development environment).